src/hotspot/share/oops/markWord.hpp

```diff
@@ -73,7 +73,7 @@ class markWord {
  public:
   explicit markWord(uintptr_t value) : _value(value) {}
 
-  markWord() { /* uninitialized */}
+  markWord() = default;         // Doesn't initialize _value.
 
   // It is critical for performance that this class be trivially
   // destructable, copyable, and assignable.
```

Comment:
Given the comment, would it make sense to also explicitly mark them as `= default`?

Reply:
Yes.  That would make it hard for someone to break the restriction without reading the comment.